src/simulate.js

```diff
@@ -28,6 +29,14 @@ export const render = (app: FusionApp) => (
   url: string,
   options: * = {}
 ): Promise<*> => {
+  if (global.jsdom) {
+    if (!url.startsWith('/')) {
+      url = `/${url}`;
+    }
+    global.jsdom.reconfigure({
+      url: `http://localhost${url}`,
```

Comment:
Is this assumption valid? Or is there a programmatic way of referencing the hostname from JSDOM?

Reply:
I am pretty certain this assumption is valid

Reply:
I was maybe concerned about having different hostnames based on url, but I now see we call reconfigure regardless of what is passed to the url parameter, so that won't happen.